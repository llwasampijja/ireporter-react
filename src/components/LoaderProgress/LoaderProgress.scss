.progress_loader {
  width: 20px;
  height: 20px;
  margin: auto;
  border-radius: 50%;
  border: 10px solid #FFFFFF;
  border-top: 10px solid #F34579;
  border-right: 10px solid #4169e1;
  border-bottom: 10px solid #F34579;
  border-left: 10px solid #4169e1;
  animation: spin 1s linear infinite;
}

@keyframes spin {
  0% { transform: rotate(0deg);}
  100% { transform: rotate(360deg);}
}
