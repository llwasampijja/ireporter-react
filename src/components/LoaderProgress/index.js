// react libraries
import React from 'react';

// styles
import './LoaderProgress.scss';

const LoaderProgress = () => <div><div className="progress_loader" /></div>;

export default LoaderProgress;
