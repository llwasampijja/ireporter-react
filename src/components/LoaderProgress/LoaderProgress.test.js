// react libraries
import React from 'react';

// third-party libraries
import { shallow } from 'enzyme';

// components
import LoaderProgress from '.';

describe('test for the LoaderProgress component', () => {
  const wrapper = shallow(<LoaderProgress />);
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
