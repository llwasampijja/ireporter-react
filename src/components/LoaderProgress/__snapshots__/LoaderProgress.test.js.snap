// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for the LoaderProgress component should match the snapshot 1`] = `
<div>
  <div
    className="progress_loader"
  />
</div>
`;
